source/index.rst

```diff
@@ -79,36 +79,43 @@ line and through an API.
    :name: toc-core
 
    parent-interactions
-   manage-roas
+   cli
+   multi-user
    monitoring
+   manage-children
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: Object Types
+   :name: object-types
+   
+   manage-roas
+   manage-aspas
+   manage-bgpsec   
 
 .. toctree::
    :maxdepth: 2
    :hidden:
    :caption: Advanced
    :name: toc-advanced
 
-   cli
-   manage-aspas
-   manage-bgpsec
-   multi-user
    ca-keyroll
    ca-migrate-repo
    publication-server
-   manage-children
    hsm
    trust-anchor
```

Comment:
Maybe move to reference? Irrelevant to most 'normal' users.

Reply:
Agreed, I doubted this one the most.